src/main/java/com/habitmon/domain/habitmon/service/HabitMonService.java

```diff
@@ -0,0 +1,26 @@
+package com.habitmon.domain.habitmon.service;
+
+import com.habitmon.domain.habit.domain.HabitMon;
+import com.habitmon.domain.habitmon.api.dto.request.HabitMonCreateRequest;
+import com.habitmon.domain.habitmon.api.dto.response.HabitMonCreateResponse;
+import com.habitmon.domain.habitmon.domain.repository.HabitMonRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class HabitMonService {
+
+    private final HabitMonRepository habitMonRepository;
+
+    public HabitMonCreateResponse createHabitMon(HabitMonCreateRequest request){
+        HabitMon habitmon = habitMonRepository.save(
+                HabitMon.builder()
+                        .name(request.name())
+                        .age(request.age())
```

Comment:
age는 상수로 넣어주면 좋을 것 같아요~!!
```suggestion
                        .age(0)
```

Reply:
네! 감사합니다